.eslintrc.js

```diff
@@ -103,7 +103,7 @@ module.exports = {
     'comma-style': ['warn', 'last'],
     'computed-property-spacing': ['warn', 'never'],
     'func-call-spacing': ['warn', 'never'],
-    indent: ['warn', 2, { SwitchCase: 0 }],
+    indent: 0,
```

Comment:
This was removed as it was/is incompatible with prettier indent rules. See [this](https://github.com/eslint/eslint/issues/15857#issuecomment-1120568763) and [this](https://stackoverflow.com/a/58977894).